### ISSUE TITLE
Streaming process fails when no new data is available

### ISSUE THREAD

**Issue by rmarabini** [scipion-em/scipion-em#91]
Version: scipion3

I have been running streaming projects with the workflow

import movies -> motioncor -> CTFfind -> monitor summary

After a while, when all the data is processed, the workflow fails showning the same error in all protocols 

Process XXXXXX not found running on the machine. It probably has died or been killed without reporting the status to Scipion. Logs might have information about what happened to this process.

By the way, logs have no information.

**Comment by pconesa** [comment]
@dmaluenda , do you think this is still happening?

**Comment by dmaluenda** [comment]
TBH, I don't know but I will check it.

Thanks

**Comment by rmarabini** [comment]
Yes, It happens to my. All data is processed but the protocol end with an
error (pink box) instead of success (green box)

**Comment by pconesa** [comment]
Is this through ssh?
I'm testing locally, scipion3 template and chose the demo and it works fine?

**Comment by dmaluenda** [comment]
Same for me. I can't reproduce this.

Cheers,
<http://ucm.es/>

*_____Dr. David Maluenda Niubó*
dmaluenda@cnb.csic.es - (+34) 619 029 310
CSIC - Centro Nacional de Biotecnología <http://www.cnb.csic.es>
BioComputing Unit <http://biocomputingunit.es/>

This email message and any documents attached to it may contain
confidential or legally protected material and are intended solely for the
use of the individual or organization to whom they are addressed. We remind
you that if you are not the intended recipient of this email message or the
person responsible for processing it, then you are not authorized to read,
save, modify, send, copy or disclose any of its contents. If you have
received this email message by mistake, we kindly ask you to inform the
sender of this and to eliminate both the message and any attachments it
carries from your account. Thank you for your collaboration.
